lib/ret.ex

```diff
@@ -40,4 +40,8 @@ defmodule Ret do
   defp valid_email_address?(string) do
     string =~ ~r/\S+@\S+/
   end
+
+  @spec turkey_mode? :: boolean
+  def turkey_mode?,
+    do: Application.fetch_env!(:ret, :turkey_mode?)
```

Comment:
This is a nitpick, but would you mind moving this function before `valid_email_address?/1` so these public functions are in alphabetical order?

Reply:
done

Reply:
Thank you!